FHICORC.Core/Services/Model/Converter/ISO8601DateTimeConverter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace FHICORC.Core.Services.Model.Converter
+{
+    /*
+     * Custom DateTime converter to support dates without days and months
+     */
+
+    public class ISO8601DateTimeConverter : DateTimeConverterBase
```

Comment:
If possible, maybe this class could have some tests? Implicitly tested in CredentialSubjectTests, but nice to have some specific tests 🙂

Reply:
I agree, please add some tests for this one.